app/workers/pr_mergeability_checker.rb

```diff
@@ -27,20 +31,29 @@ def process_mergeability
       write_to_github
       apply_label
     elsif !was_mergeable && currently_mergeable
+      remove_comments
       remove_label
     end
 
     # Update columns directly to avoid collisions wrt the serialized column issue
     branch.update_columns(:mergeable => currently_mergeable)
   end
 
+  def remove_comments
+    comment_ids = GithubService.issue_comments(fq_repo_name, branch.pr_number).select do |com|
+      com.user.login == Settings.github_credentials.username && com.body.eql?(unmergeable_comment)
```

Comment:
Only look for the leading tag and not the entire string.  The purpose of the tag is to allow us to change the string at any time while still allowing the comment finder to work.